links.html

```diff
@@ -55,7 +55,7 @@ <h1>Useful Links </h1>
             </blockquote>
             <p>&copy; 2019 <a href="mailto:carlin.linklater@student.saultcollege.ca?subject=Message%20from%20CSD120%20Website">Carlin Linklater</a>            
         <a href="images/sc-logo.png"><img src="images/sc-logo.png"
-        sizes= 10.6vw
+        sizes="10.6vw"

```

Comment:
It's best practice to always quote your attribute values, even if it is valid.  The only reason not to would be if you're trying really hard to shave off bandwidth